utilsplot.py

```diff
@@ -573,6 +573,18 @@ def rga_plot(G, w_start=-2, w_end=2, axlim=None, points=1000, fig=0,
     freqresp = [G(si) for si in s]
 
     plot_No = 1
+    
```

Comment:
Please make sure your syntax conforms to PEP8. The spacing around this dictionary is wonky.

Reply:
**Addressed.**